src/mapeo-project.js

```diff
@@ -124,4 +166,34 @@ export class MapeoProject {
   get field() {
     return this.#dataTypes.field
   }
+
+  /**
+   * @param {Omit<import('@mapeo/schema').ProjectValue, 'schemaName'>} settings
+   * @param {string} [versionId]
+   * @returns {Promise<import('@mapeo/schema').Project>}
+   */
+  async $setProjectSettings(settings, versionId) {
```

Comment:
does it make sense for this to return the whole updated document, or just a subset? maybe something similar to `$getProjectSettings`?

Reply:
decided to return just a subset (see `EditableSettings` type)